R/plot_section.R

```diff
@@ -0,0 +1,153 @@
+#' Generate stratigraphic section plot
+#'
+#' A function to plot the stratigraphic ranges of fossil taxa from occurrence
+#' data.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences containing
+#' at least two columns: names of taxa, and their stratigraphic position
+#' (see `name` and `level` arguments).
+#' @param name \code{character}. The name of the column you wish to be treated
+#' as the input names, e.g. "genus" (default).
+#' @param level \code{character}. The name of the column you wish to be treated
+#' as the stratigraphic levels associated with each occurrence, e.g. "bed"
+#' (default) or "height".
+#' @param certainty \code{character}. The name of the column you wish to be
+#' treated as the information on whether an identification is certain ("1") or
+#' uncertain ("0"). By default, no column name is provided, and all occurrences
+#' are assumed to be certain. In the plot, any occurrence labelled as certain
+#' will be plotted with a black circle, while any occurrence labelled as
+#' uncertain will be plotted with a white circle.
+#' @param by \code{character}. How should the output be sorted?
+#' Either: "FAD" (first appearance; default), "LAD" (last appearance), or
+#' "name" (alphabetically by taxon names).
+#' @param units \code{character}. Should the y-axis be labelled as "height"
```

Comment:
I am not sure we really need this parameter (or label), we could just use the default plot parameters available (main, ylab, etc) which could be passed via `...`.